src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberSupplier {
+    public int getRandomNumber() {
+        return new Random().nextInt(100);
+    }
+}
```

Comment:
The use of a new Random instance each time the getRandomNumber() method is called is not efficient. Instead, you should create a single Random instance as a class-level variable and reuse it for generating numbers.